### PR TITLE
tikv: document periodic full compaction

### PR DESCRIPTION
**Pull request by Oreoxmt**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Document periodic-full-compact-start-times and periodic-full-compact-start-max-cpu
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/15630
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1030,6 +1030,16 @@ raftstore 相关的配置项。
 + 默认值：0.1
 + 最小值：0
 
+### `periodic-full-compact-start-times` <span class="version-mark">从 v7.6.0 版本开始引入</span>
+
++ 设置启动周期性全量数据整理 (Compaction) 的时间。你可以在数组中指定多个时间计划。例如，`periodic-full-compact-start-times = ["03:00", "23:00"]` 表示 TiKV 基于 TiKV 节点的本地时区，在凌晨 3 点和晚上 11 点进行全量数据整理。
```
Comment (qiancai) [inline tikv-configuration-file.md:1035]: 是每天吗

Comment (qiancai) [inline tikv-configuration-file.md:1035]:
```suggestion
+ 设置 TiKV 启动周期性全量数据整理 (Compaction) 的时间。你可以在数组中指定一个或多个时间计划。例如，`periodic-full-compact-start-times = ["03:00", "23:00"]` 表示 TiKV 基于 TiKV 节点的本地时区，在凌晨 3 点和晚上 11 点进行全量数据整理。
```

Comment (tonyxuqqi) [reply tikv-configuration-file.md:1035]: 这里标注一下是实验特性吧

Comment (tonyxuqqi) [reply tikv-configuration-file.md:1035]: @qiancai 是每天

Comment (tonyxuqqi) [reply tikv-configuration-file.md:1035]: 可以加一个带时区的例子比如["23:00 +0700", "03:00 +0700"]

**Review by tonyxuqqi — APPROVED** [review]
Rest LGTM

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1030,6 +1030,16 @@ raftstore 相关的配置项。
 + 默认值：0.1
 + 最小值：0
 
+### `periodic-full-compact-start-times` <span class="version-mark">从 v7.6.0 版本开始引入</span>
+
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:1034]:
```suggestion

> **警告：**
>
> 周期性全量数据整理目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。

```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -1030,6 +1030,20 @@ raftstore 相关的配置项。
...
+> **警告：**
+>
+> 周期性全量数据整理目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
+
++ 设置 TiKV 启动周期性全量数据整理 (Compaction) 的时间。你可以在数组中指定一个或多个时间计划。例如，`periodic-full-compact-start-times = ["03:00", "23:00"]` 表示 TiKV 基于 TiKV 节点的本地时区，在每天凌晨 3 点和晚上 11 点进行全量数据整理。
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:1039]:
```suggestion
+ 设置 TiKV 启动周期性全量数据整理 (Compaction) 的时间。你可以在数组中指定一个或多个时间计划。例如，`periodic-full-compact-start-times = ["03:00", "23:00"]` 表示 TiKV 基于 TiKV 节点的本地时区，在每天凌晨 3 点和晚上 11 点进行全量数据整理。`periodic-full-compact-start-times = ["03:00 +0000", "23:00 +0000"]` 表示 TiKV 在每天 UTC 时间的凌晨 3 点和晚上 11 点进行全量数据整理。
```
@tonyxuqqi PTAL

**Review by qiancai — APPROVED** [review]

**Pull request merged**